flocker_bb/builders/flocker_vagrant.py

```diff
@@ -447,6 +447,8 @@ def slave_class(self):
             return 'aws/centos-7'
 
 
+TUTORIAL_DISTRIBUTION = "centos-7"
```

Comment:
I was wondering if it would make sense to have this be like the other `*_CONFIGURATIONS`. But I guess as long as we plan to only have one tutorial box, that probably isn't worth it.